website/src/docs/aws-s3-multipart.md

```diff
@@ -40,11 +40,19 @@ The `@uppy/aws-s3-multipart` plugin has the following configurable options:
 
 ### `limit: 0`
 
-The maximum amount of chunks to upload simultaneously. Set to `0` to disable limiting.
+The maximum amount of chunks to upload simultaneously.
```

Comment:
I removed the comment "Set to `0` to disable limiting."` here because it is not true -- the limit defaults to 5 here:

https://github.com/martin-brennan/uppy/blob/af9030ece598bc2cb7835100b165c7cd9736c616/packages/@uppy/aws-s3-multipart/src/index.js#L218-L218

Also I don't think the candidate part gathering code would work with no limit?